content/sensu-core/2.0/getting-started/recommended-hardware.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Hardware Requirements and Recommendations"
+linkTitle: "Hardware Requirements"
+description: "Guidelines for what you'll need for deploying Sensu Core to production."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+platforms: ["Ubuntu/Debian", "RHEL/CentOS", "Windows"]
+menu:
+  sensu-core-2.0:
+    parent: getting-started
+---
+
+### Backend Minimum Requirements
+
+* 64-bit Intel or AMD CPU
+* 4 GB RAM
```

Comment:
Out of curiosity, where this number comes from? A first glance, it seems a lot but I'm probably missing something.

Reply:
It's a compromise between my load testing observations and this passage from the etcd docs:
```
etcd has a relatively small memory footprint but its performance still depends on having enough 
memory. An etcd server will aggressively cache key-value data and spends most of the rest of its
memory tracking watchers. Typically 8GB is enough. For heavy deployments with thousands of
watchers and millions of keys, allocate 16GB to 64GB memory accordingly.
```